website/docs/docs/build/conversion-metrics.md

```diff
@@ -0,0 +1,352 @@
+---
+title: "Conversion metrics"
+id: conversion
+description: "Use Conversion metrics to measure conversion events."
+sidebar_label: Conversion
+tags: [Metrics, Semantic Layer]
+---
+
+Conversion metrics allow us to define when a base event happens and a subsequent conversion event happens for a specific entity within some time range.
+
+- **Example**: Track how often a user (entity) completes a visit (base event) and then makes a purchase (conversion event) within 7 days (time window).
+- **Requirements**: A time range and an entity to join on.
+- **How it differs from [Ratio metrics](/docs/build/ratio)**: Includes an entity in the pre-aggregated join.
```

Comment:
jordan originally did have it as a paragraph without bullets but when i read it, i felt it was a lot of info at first glance. since this is a new metric type we're introducing, i chose to break it down further for the user. 

![Screenshot 2024-01-10 at 17 12 23](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/a98bf221-dfa7-47ed-9daa-fb7d3717c00f)

Reply:
meeting you half way and remoing bullets but adding another line. + make it succinct